core/src/io/github/teamfractal/entity/Player.java

```diff
@@ -257,28 +257,21 @@ public void sellResourceToMarket(int amount, Market market, ResourceType resourc
      * @param plot           The landplot to purchase
      */
```

Comment:
Coulp add throws documenation for the NotEnoughMoneyException. Dont need to both with null pointer exception as thats pretty obvious